tests/acceptance/features/loginUsingEmailAddress.feature

```diff
@@ -0,0 +1,41 @@
+@insulated @disablePreviews
+Feature: login users
+
+As an admin
+I want users to login with their email address stored on a LDAP server
+So that they only need to remember one username and password (SSO)
+
+	Scenario: login with default settings
+		When the LDAP users have been resyncted
+		Then it should be possible to login with the username "user1@example.org" and password "1234" into the WebUI
+
+	Scenario: using ldap filter including email field 
+		When the LDAP config "LDAPTestId" has these settings:
+		|key                | value                                                                                    |
+		|ldapLoginFilter    | (&(\|(objectclass=inetOrgPerson))(\|(uid=%uid)(\|(mailPrimaryAddress=%uid)(mail=%uid)))) |
```

Comment:
The bracketing is suboptimal
`(&(|(objectclass=inetOrgPerson))(|(uid=%uid)(|(mailPrimaryAddress=%uid)(mail=%uid))))` should be
`(&(objectclass=inetOrgPerson)(|(uid=%uid)(mailPrimaryAddress=%uid)(mail=%uid)))`